# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bsd-games/delay_output-sym.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

size of symbol `delay_output' changed from 21 in hack/hack.termcap.o to 161
in libncurses.a

diff -urN -x '*~' bsd-games-2.14/hack/extern.h bsd-games-2.14-fix/hack/extern.h
--- ./hack/extern.h.orig	2004-01-27 17:51:11.000000000 -0300
+++ ./hack/extern.h	2005-03-01 00:53:42.691752816 -0300
@@ -511,7 +511,7 @@
 void standoutend(void);
 void backsp(void);
 void bell(void);
-void delay_output(void);
+void delay_output_my(void);
 void cl_eos(void);
 
 /* hack.timeout.c */
diff -urN -x '*~' bsd-games-2.14/hack/hack.pri.c bsd-games-2.14-fix/hack/hack.pri.c
--- bsd-games-2.14/hack/hack.pri.c	2003-03-29 13:05:30.000000000 +0100
+++ bsd-games-2.14-fix/hack/hack.pri.c	2003-10-02 03:56:22.000000000 +0200
@@ -171,7 +171,7 @@
 		return;
 	}
 	if (prevx >= 0 && cansee(prevx, prevy)) {
-		delay_output();
+		delay_output_my();
 		prl(prevx, prevy);	/* in case there was a monster */
 		at(prevx, prevy, levl[prevx][prevy].scrsym);
 	}
diff -urN -x '*~' bsd-games-2.14/hack/hack.termcap.c bsd-games-2.14-fix/hack/hack.termcap.c
--- bsd-games-2.14/hack/hack.termcap.c	2003-03-29 13:05:30.000000000 +0100
+++ bsd-games-2.14-fix/hack/hack.termcap.c	2003-10-02 03:56:30.000000000 +0200
@@ -326,7 +326,7 @@
 }
 
 void
-delay_output()
+delay_output_my()
 {
 
 	/* delay 50 ms - could also use a 'nap'-system call */
